lib/tracer/jaeger.go

```diff
@@ -20,7 +20,9 @@ func init() {
 		Summary: `
 Send spans to a [Jaeger](https://www.jaegertracing.io/) agent.`,
 		FieldSpecs: docs.FieldSpecs{
-			docs.FieldCommon("agent_address", "The address of a Jaeger agent to send tracing events to."),
+			docs.FieldCommon("agent_address", "The address of a Jaeger agent to send tracing events to.", "jaeger-agent:6831"),
+			docs.FieldCommon("collector_address", "The address of a Jaeger collector to send tracing events to. If set, this will override `agent_address`.",
```

Comment:
Looks like this field accepts full URLs so it should be called `collector_url` in order to fit in with other Benthos fields.